CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to Harmony-Py
+
+Thanks for contributing!
+
+## Making Changes
+
+To allow us to incorporate your changes, please use the following process:
+
+1. Fork this repository to your personal account.
+2. Create a branch and make your changes.
+3. Test the changes locally/in your personal fork.
+4. Submit a pull request to open a discussion about your proposed changes.
+5. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+## Commits
+
+Our ticketing and CI/CD tools are configured to sync statuses amongst each other. Commits play an important role in this process. Please start all commits with the Harmony ticket number associated with your feature, task, or bug. All commit messages should follow the format "harmony-XXXX: [Your commit message here]"
```

Comment:
minor nit - looks like we usually use all caps in our commit messages to match JIRA - HARMONY-XXX